task_7.py

```diff
@@ -0,0 +1,5 @@
+def genFib(n):
+    a, b = 0, 1
+    for i in range(n + 1):
+        yield a
+        print(list(genFib(5)))
```

Comment:
This print shouldn't be here. You made is as an endless recursion and it won't work. 
Also, you should write a construction of finding `n` element of the sequence. You can do it by transforming your generator to a list and getting the last element of the list `list(genFib(5))[-1]`